core/src/components/item-divider/test/basic/item-divider.e2e.ts

```diff
@@ -46,5 +46,32 @@ configs().forEach(({ title, screenshot, config }) => {
       const divider = page.locator('ion-item-divider');
       await expect(divider).toHaveScreenshot(screenshot(`item-divider-icon-start`));
     });
+
+    /**
+     * This behavior needs to be tested for all modes & directions
+     * Safe padding should stay on the same side when the direction changes
+     */
+    test('should have safe area padding', async ({ page }) => {
+      await page.setContent(
+        `
+        <style>
+          :root {
+            --ion-safe-area-left: 40px;
+            --ion-safe-area-right: 20px;
+          }
+        </style>
+        <ion-list>
+          <ion-item-divider>
+            <ion-label>Item Divider</ion-label>
```

Comment:
```suggestion
            <ion-label>Item Divider</ion-label>
            <ion-button slot="end">Button</ion-button>
```

Could we add a button to the test to further verify the safe area is working when flipping?